[2023-11-21T16:43:30.750+0545] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: podcast_summery.download_episodes manual__2023-11-21T10:58:20.700564+00:00 [queued]>
[2023-11-21T16:43:30.754+0545] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: podcast_summery.download_episodes manual__2023-11-21T10:58:20.700564+00:00 [queued]>
[2023-11-21T16:43:30.755+0545] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-21T16:43:30.767+0545] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): download_episodes> on 2023-11-21 10:58:20.700564+00:00
[2023-11-21T16:43:30.770+0545] {standard_task_runner.py:57} INFO - Started process 219948 to run task
[2023-11-21T16:43:30.773+0545] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'podcast_summery', 'download_episodes', 'manual__2023-11-21T10:58:20.700564+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/podcast_summery.py', '--cfg-path', '/tmp/tmpzkbj2vj0']
[2023-11-21T16:43:30.774+0545] {standard_task_runner.py:85} INFO - Job 64: Subtask download_episodes
[2023-11-21T16:43:30.801+0545] {task_command.py:416} INFO - Running <TaskInstance: podcast_summery.download_episodes manual__2023-11-21T10:58:20.700564+00:00 [running]> on host dwarka.dwarka
[2023-11-21T16:43:30.854+0545] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='podcast_summery' AIRFLOW_CTX_TASK_ID='download_episodes' AIRFLOW_CTX_EXECUTION_DATE='2023-11-21T10:58:20.700564+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-21T10:58:20.700564+00:00'
[2023-11-21T16:43:30.855+0545] {logging_mixin.py:154} INFO - episodes/whats-next-for-the-artificial-intelligence-industry.mp3
[2023-11-21T16:43:30.855+0545] {logging_mixin.py:154} INFO - Downloading whats-next-for-the-artificial-intelligence-industry.mp3
[2023-11-21T16:44:00.150+0545] {process_utils.py:131} INFO - Sending 15 to group 219948. PIDs of all processes in the group: [219948]
[2023-11-21T16:44:00.150+0545] {process_utils.py:86} INFO - Sending the signal 15 to group 219948
[2023-11-21T16:44:00.151+0545] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-11-21T16:44:00.160+0545] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/airflow/dags/podcast_summery.py", line 65, in download_episodes
    audio=requests.get(episode["enclosure"]["@url"])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/sessions.py", line 725, in send
    history = [resp for resp in gen]
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/sessions.py", line 725, in <listcomp>
    history = [resp for resp in gen]
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/sessions.py", line 266, in resolve_redirects
    resp = self.send(
           ^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/sessions.py", line 747, in send
    r.content
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/models.py", line 899, in content
    self._content = b"".join(self.iter_content(CONTENT_CHUNK_SIZE)) or b""
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/requests/models.py", line 816, in generate
    yield from self.raw.stream(chunk_size, decode_content=True)
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/urllib3/response.py", line 934, in stream
    data = self.read(amt=amt, decode_content=decode_content)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/urllib3/response.py", line 877, in read
    data = self._raw_read(amt)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/urllib3/response.py", line 812, in _raw_read
    data = self._fp_read(amt) if not fp_closed else b""
           ^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/urllib3/response.py", line 797, in _fp_read
    return self._fp.read(amt) if amt is not None else self._fp.read()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/http/client.py", line 466, in read
    s = self.fp.read(amt)
        ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/socket.py", line 706, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/ssl.py", line 1311, in recv_into
    return self.read(nbytes, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/ssl.py", line 1167, in read
    return self._sslobj.read(len, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ace/aiflow_project/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 1634, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-11-21T16:44:00.171+0545] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=podcast_summery, task_id=download_episodes, execution_date=20231121T105820, start_date=20231121T105830, end_date=20231121T105900
[2023-11-21T16:44:00.179+0545] {standard_task_runner.py:104} ERROR - Failed to execute job 64 for task download_episodes (Task received SIGTERM signal; 219948)
[2023-11-21T16:44:00.203+0545] {process_utils.py:79} INFO - Process psutil.Process(pid=219948, status='terminated', exitcode=1, started='16:43:30') (219948) terminated with exit code 1
